packages/experimental/src/smart-session/controllers/SmartSessionsController.ts

```diff
@@ -224,15 +226,11 @@ export const SmartSessionsController = {
         pci: session.pci,
         permissions: [...session.permissions.map(p => JSON.parse(JSON.stringify(p)))],
         expiry: Math.floor(session.expiry / 1000),
-        address: activeCaipAddress as `0x${string}`
+        address: chainAndAddress.address
```

Comment:
Found here that `revokePermissions` expect address value but we are passing CAIP address, so this should be the right one. But maybe cannot see something @tomiir this is an old code would like to validate with u if we should pass CAIP address or Hex value?